src/index.html

```diff
@@ -1,21 +1,70 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta 
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
       "
     >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+
+      <div class="container__card card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <h2 class="card__product">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews-count">
+            Reviews: 5
+          </div>
+
+        </div>
+
+        <div class="card__cost">
+          <div class="card__cost-text">
+            Price:
+          </div>
+
+          <div class="card__cost-amount">
+            $2,199
+          </div>
+        </div>
+
+        <a
+          class="card__button"
+          data-qa="hover"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
+    </div>
   </body>
-</html>
+</html>
```

Comment:
<img width="254" alt="image" src="https://user-images.githubusercontent.com/99403198/232007310-e3451f45-0f45-4549-949f-3d1a370bd9a3.png">

```suggestion
</html>

```